log/logger_test.go

```diff
@@ -132,12 +120,12 @@ func testLogger(t *testing.T, context spec.G, it spec.S) {
 
 		it("writes debug log", func() {
 			l.Debug("test-message")
-			Expect(b.String()).To(Equal("test-message\n"))
+			Expect(b.String()).To(Equal("\x1b[46mtest-message\x1b[0m\n"))
```

Comment:
what's `\x41b` and why is it there?

Reply:
It's color/placement terminal codes. Previously, debug output was just plain text. It stood out against other stuff we logged. Since we can now control the entire logger in libcnb v2, I've changed this to make debug output fit in with the rest of our output.

Reply:
cool! readability FTW!